### DISCUSSION TITLE
Improve "View Configuration" & "Open in Launchpad" actions for Schedules

### DISCUSSION THREAD

**Discussion by cbini** [dagster-io/dagster#7724] [Ideas]
If you set up a Schedule for a Job that has a default set of config and tags, and you pass a different set config/tags to the Schedule, it will merge with the default config/tags at runtime as expected. 

However, if you look under Runs > Scheduled and use the "View Configuration" action, it will only display the Schedule config/tags, not the merged version that will ultimately run.

Similarly, the "Open in Launchpad" action, will display only the Schedule config with only the preset Job tags.

Essentially, there's no way of previewing the config/tags that will run for a future Schedule if it has it's own config and tag set. Not sure if this is a feature or a bug, but either way it would be helpful to at least have the option to view the complete set of parameters that the Schedule will use at runtime.

**Comment by yuhan** [comment]
Locking this discussion as we are moving the conversation to https://github.com/dagster-io/dagster/issues/7814
